### ISSUE TITLE
Screenreader Accessibility...

### ISSUE THREAD

**Issue by mekura** [oakmac/chessboardjs#126]
At present no alt text is added to pieces to denote color or piece type as well as what square they are on. At present screenreader software ignores these images.
